### DISCUSSION TITLE
Does it work with Kicad 7?

### DISCUSSION THREAD

**Discussion by rufsiany-sr** [bennymeg/Fabrication-Toolkit#47] [Q&A]
H, I've added 'LCSC Part #' field to symbol properties list in the sch file (and put the appropriate JLC part numbers), but in the 'bom.csv' generated file this field's column is empty. I've tried also with other names (e.g.: MPN), but no matter of which field name I put, the column with JLC part numbers always stays empty. What am I doing wrong?

**Comment by romain145** [comment]
A late answer, but don't forget to go in the PCB Editor and update the PCB with the changes made to the schematic.
The JLC Plugin works from the PCB, not the schematic.
